models/ChangePasswordForm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace app\models;
+
+use Yii;
+use yii\base\Model;
+
+/**
+ * @property User $user
+ */
+class ChangePasswordForm extends Model
+{
+    /**
+     * @var string
+     */
+    public $currentPassword;
+    /**
+     * @var string
+     */
+    public $password;
+    /**
+     * @var string
+     */
+    public $passwordRe;
```

Comment:
It is better not to trim variable names like that. `passwordRepeat`?

Reply:
Done.